modules/packages/UnitTest.chpl

```diff
@@ -401,19 +401,19 @@ module UnitTest {
         tmpString = seq_type_name+"s differ: ";
         if seq_type_name == "Array" {
           tmpString += "'[";
-          for i in 1..seq1.size {
+          for i in seq1.indices {
             if i != seq1.size then tmpString+= seq1[i]:string+", ";
             else tmpString += seq1[i]:string+"]' != '[";
           }
-          for i in 1..seq2.size {
+          for i in seq2.indices {
             if i != seq2.size then tmpString+= seq2[i]:string+", ";
             else tmpString += seq2[i]:string+"]'";
           }
         }
         else {
           tmpString += "'"+stringify(seq1)+"' != '"+stringify(seq2)+"'" ;
         }
-        for i in 1..min(len1,len2) {
+        for i in seq1.indices.low..min(len1,len2) {
```

Comment:
Can this be sure both sequences start with the same index? `seq1.indices[seq2.indices]` would give just the intersection if they did differ.

Reply:
Good point. I'll see if I can write a test that fails for this and then modify the implementation to fix.

Thanks for the review!

Reply:
This code is actually unused since array equality is done by reduction now. I have removed this code and added a test for this case.